src/common/signalk/SKNMEA2000Parser.cpp

```diff
@@ -43,19 +46,170 @@ const SKUpdate& SKNMEA2000Parser::parse(const SKSourceInput& input, const tN2kMs
+    //case 127257: // Attitude
+    //case 127488: // Engine parameters rapid
+    //case 127493: // Transmission parameters: dynamic
+    //case 127501: // Binary status report
+    //case 127505: // Fluid level
+    //case 127508: // Battery Status
+    //case 127513: // Battery Configuration Status
+    //case 129025: // Lat/lon rapid
+    //case 129283: // Cross Track Error
+    //case 130310: // Outside Environmental parameters
+    //case 130311: // Environmental parameters
+    //case 130312: // Temperature
+    //case 130314: // Pressure
+    //case 130316: // Temperature extended range
+    default:
+      DEBUG("No known conversion for PGN %i", msg.PGN);
+      return _invalidSku;
+  } // end switch msg.PGN
+}
+
+// System Time / Date
+// TODO: Set system date and time for SD-Card logging file date and
+// Offset to millis() for recalculation of true date/time
+// TODO: after 42 days millis() overflow, new time sync needed
+const SKUpdate& SKNMEA2000Parser::parse126992(const SKSourceInput& input, const tN2kMsg& msg, const SKTime& timestamp) {
+  unsigned char sid;          // Sequence ID
+  uint16_t SystemDate;        // Days since 1970-01-01
+  double SystemTime;          // seconds since midnight mit 2 Kommastellen
+  tN2kTimeSource TimeSource;  // see tN2kTimeSource (z.B.: GPS)
+
+    if (ParseN2kSystemTime(msg,sid,SystemDate,SystemTime,TimeSource) ) {
+
+      /*
+      // static global variables needed....
+      if ( SystemDate > 0 && SystemTime > 0 && dateTime::timeStampNeeded ) {
+        // time function class
+        dt.setTimeStamp( &SystemDate, &SystemTime);
+        // the first 20 Seconds try to get date and time from N2k bus
+        if ( millis() > 20000 ) dateTime::timeStampNeeded = false;
+      }
+      */
+    } else {
+      INFO("Unable to parse NMEA2000 with PGN %i", msg.PGN);
+    }
+  // nothing todo at the moment
+  return _invalidSku;
+}
+
+//  ***********************************************
+//   PGN 127245 Rudder
+//        →  sid
+//        →  RudderPosition [rad]
+// *  ********************************************** */
+const SKUpdate& SKNMEA2000Parser::parse127245(const SKSourceInput& input, const tN2kMsg& msg, const SKTime& timestamp) {
+  unsigned char Instance;
+  tN2kRudderDirectionOrder RudderDirectionOrder;
+  double RudderPosition;
+  double AngleOrder;
+
+  if (ParseN2kRudder(msg,RudderPosition,Instance,RudderDirectionOrder,AngleOrder) ) {
+    SKUpdateStatic<2> *update = new SKUpdateStatic<2>();
+    update->setTimestamp(timestamp);
+
+    SKSource source = SKSource::sourceForNMEA2000(input, msg.PGN, msg.Priority, msg.Source);
+    update->setSource(source);
+    update->setSteeringRudderAngle(RudderPosition);
+    DEBUG("Rudder: %d",SKRadToDeg(RudderPosition));
+    _sku = update;
+    return *_sku;
   }
-  // Water Depth
-  if (msg.PGN == 128267) {
-    return parse128267(input, msg, timestamp);
+  else {
+    INFO("Unable to parse NMEA2000 with PGN %i", msg.PGN);
```

Comment:
I think this should be a `DEBUG` because it could get repeated a lot if you are on a network where something is sending a PGN we do not understand. 

The theory is that INFO message should be non-verbose enough that they can be kept on all the time and even logged to a file.